styles/scss/image-gallery.scss

```diff
@@ -115,15 +112,6 @@ $ig-transparent: rgba(0, 0, 0, 0) !default;
     }
   }
 
-  &:hover::before {
-    color: $ig-white;
-    transform: scale(1.1);
-
-    @media (max-width: $ig-screen-sm-min) {
-      transform: none;
-    }
-  }
-
```

Comment:
This was removed, because of the above changes, which essentially superseded it.
The original code apparently overrode the blue coloring in the above CSS, putting it back to white, but added a 10% scale up; it also didn't include a focus state for keyboard users.
Wrapping it in with the above section ensured all the primary interactive components responded the same way to the mouse and keyboard interaction.
For the same reason as I removed going back to white on small screens, I removed dropping of the scaling because it didn't make sense to me, but I may just not understand it.
Not sure they add much, but screenshots are being provided to show how this and the above changed things.
<img width="237" alt="Screen Shot 2019-11-08 at 12 18 43 PM" src="https://user-images.githubusercontent.com/19862719/68497388-6e3f3080-0222-11ea-9270-01c8e2633718.png">
<img width="391" alt="Screen Shot 2019-11-08 at 12 17 59 PM" src="https://user-images.githubusercontent.com/19862719/68497403-75fed500-0222-11ea-86a7-d2a8e3fc3f86.png">